src/blob_storage.cc

```diff
@@ -24,6 +24,26 @@ Status BlobStorage::NewPrefetcher(uint64_t file_number,
                                     result);
 }
 
+Status BlobStorage::DeleteBlobsInRanges(const RangePtr* ranges, size_t n,
+                                        bool include_end,
+                                        SequenceNumber obsolete_sequence) {
+  MutexLock l(&mutex_);
+  for (size_t i = 0; i < n; i++) {
+    auto begin = ranges[i].start, end = ranges[i].limit;
```

Comment:
One variable per line? 

Also change `auto` to `Slice*`? For simpler types I prefer writing out the type instead of `auto` for readability.